tests/load/load_test_all_read_setup_endpoints.sh

```
@@ -0,0 +1,540 @@
+# Test configuration
+REQUESTS=${REQUESTS:-50}
+CONCURRENCY=${CONCURRENCY:-10}
+
+# Report files (always overwritten)
+JSON_REPORT="$SCRIPT_DIR/api_test_report.json"
+HTML_REPORT="$SCRIPT_DIR/api_test_report.html"
+
+# Test data
+TEST_ADDRESS="0x70997970C51812dc3A010C7d01b50e0d17dc79C8"
+TEST_ADDRESS_2="0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC"
+TEST_PRIVATE_KEY="0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d"
```

Shellcheck (0.10.0)

[warning] TEST_PRIVATE_KEY appears unused. Verify use (or export if used externally).

(SC2034)